src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.mux.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Windows.UI.Xaml.Controls
+{
+	public partial class TextBlock
+	{
+		internal void SetUseHyperlinkForegroundOnBackPlate(bool useHyperlinkForegroundOnBackPlate)
+		{
+			// TODO Uno: BackPlate on TextBlock is not supported.
```

Comment:
Should we log it? raise a recoverable exception?

Reply:
I would suggest not logging this, as it is called on every measure. The TODO is there for `HyperlinkButton`. It should not be a critical functionality.